app.rb

```diff
@@ -39,6 +44,20 @@
   def current_user
     @current_user ||= User[session[:user_id]]
   end
+
+  def legislature_for_file(path)
+    dirs = path.split('/')
+    return nil unless dirs.first == 'data'
+    return dirs[1,2].join('/')
+  end
+
+  def legislatures_in_pr(payload)
+    pr_files = Octokit.pull_request_files(payload['repository']['full_name'], payload['number'])
+    included_legislatures = Set.new
+    included_legislatures.add(
+      pr_files.map { |f| legislature_for_file(f['filename']) }.compact
+    ).to_a
```

Comment:
Couldn't this now just be as simple as `included_legislatures = pr_files.map { |f| legislature_for_file(f['filename']) }.compact.uniq` (with an added `.to_set` if you need that, but I don't think you do)? (with actually no need to even assign it to a variable as you're just returning it straightaway)?